[1] Burnett, G. and Kennedy, E.P. The enzymatic phosphorylation of proteins. J Biol Chem. 1954 Dec;211(2):969-80. [PubMed:13221602].
[2] Walsh, D.A., Perkins, J.P. and Krebs, E.G. An adenosine 3’,5’-monophosphate-dependant protein kinase from rabbit skeletal muscle. J Biol Chem. 1968 Jul 10;243(13):3763-5. [PubMed:4298072].
[3] Ubersax, J.A. and Ferrell, J.E., Jr. Mechanisms of specificity in protein phosphorylation. Nat Rev Mol Cell Biol. 2007 Jul;8(7):530-41. [PubMed:17585314] [doi:10.1038/nrm2203].
[4] Sharma, K., D’Souza, R.C., Tyanova, S., Schaab, C., Wisniewski, J.R., Cox, J. and Mann, M. Ultradeep human phosphoproteome reveals a distinct regulatory nature of Tyr and Ser/Thr-based signaling. Cell Rep. 2014 Sep 11;8(5):1583-94. [PubMed:25159151] [doi:10.1016/j.celrep.2014.07.036].
[5] Manning, G., Whyte, D.B., Martinez, R., Hunter, T. and Sudarsanam, S. The protein kinase complement of the human genome. Science. 2002 Dec 6;298(5600):1912-34. [PubMed:12471243] [doi:10.1126/science.1075762].
[6] Rowley, J.D. Letter: A new consistent chromosomal abnormality in chronic myelogenous leukaemia identified by quinacrine fluorescence and Giemsa staining. Nature. 1973 Jun 1;243(5405):290-3. [PubMed:4126434] [doi:10.1038/243290a0].
[7] Collins, S.J. and Groudine, M.T. Rearrangement and amplification of c-abl sequences in the human chronic myelogenous leukemia cell line K-562. Proc Natl Acad Sci U S A. 1983 Aug;80(15):4813-7. [PubMed:6308652] [PubMed Central:PMC384135] [doi:10.1073/pnas.80.15.4813].
[8] George, S., Rochford, J.J., Wolfrum, C., Gray, S.L., Schinner, S., Wilson, J.C., Soos, M.A., Murgatroyd, P.R., Williams, R.M., Acerini, C.L. et al. A family with severe insulin resistance and diabetes due to a mutation in AKT2. Science. 2004 May 28;304(5675):1325-8. [PubMed:15166380] [doi:10.1126/science.1096706].
[9] Alsina-Sanchis, E., Garcia-Ibanez, Y., Figueiredo, A.M., Riera-Domingo, C., Figueras, A., Matias-Guiu, X., Casanovas, O., Botella, L.M., Pujana, M.A., Riera-Mestre, A. et al. ALK1 Loss Results in Vascular Hyperplasia in Mice and Humans Through PI3K Activation. Arterioscler Thromb Vasc Biol. 2018 May;38(5):1216-1229. [PubMed:29449337] [doi:10.1161/ATVBAHA.118.310760].
[10] White, M.J., Morris, C.P., Lawford, B.R. and Young, R.M. Behavioral phenotypes of impulsivity related to the ANKK1 gene are independent of an acute stressor. Behav Brain Funct. 2008 Nov 24;4:54. [PubMed:19025655] [PubMed Central:PMC2607297] [doi:10.1186/1744-9081-4-54].
[11] Mann, M., Ong, S.-E., Grønborg, M., Steen, H., Jensen, O.N. and Pandey, A. Analysis of protein phosphorylation using mass spectrometry: deciphering the phosphoproteome. Trends Biotechnol. 2002 Jun;20(6):261-8. [PubMed:12007495] [doi:10.1016/s0167-7799(02)01944-3].
[12] Casado, P., Rodriguez-Prados, J.C., Cosulich, S.C., Guichard, S., Vanhaesebroeck, B., Joel, S. and Cutillas, P.R. Kinase-substrate enrichment analysis provides insights into the heterogeneity of signaling pathway activation in leukemia cells. Sci Signal. 2013 Mar 26;6(268):rs6. [PubMed:23532336] [doi:10.1126/scisignal.2003573].
[13] Ferguson, F.M. and Gray, N.S. Kinase inhibitors: the road ahead. Nat Rev Drug Discov. 2018 May;17(5):353-377. [PubMed:29545548] [doi:10.1038/nrd.2018.21].
[14] Chen, E.Y., Xu, H., Gordonov, S., Lim, M.P., Perkins, M.H. and Ma’ayan, A. Expression2Kinases: mRNA profiling linked to multiple upstream regulatory layers. Bioinformatics. 2012 Jan 1;28(1):105-11. [PubMed:22080467] [PubMed Central:PMC3244772] [doi:10.1093/bioinformatics/btr625].
[15] Clarke, D.J.B., Kuleshov, M.V., Schilder, B.M., Torre, D., Duffy, M.E., Keenan, A.B., Lachmann, A., Feldmann, A.S., Gundersen, G.W., Silverstein, M.C. et al. eXpression2Kinases (X2K) Web: linking expression signatures to upstream cell signaling networks. Nucleic Acids Res. 2018 Jul 2;46(W1):W171-W179. [PubMed:29800326] [PubMed Central:PMC6030863] [doi:10.1093/nar/gky458].
[16] Krug, K., Mertins, P., Zhang, B., Hornbeck, P., Raju, R., Ahmad, R., Szucs, M., Mundt, F., Forestier, D., Jane-Valbuena, J. et al. A Curated Resource for Phosphosite-specific Signature Analysis. Mol Cell Proteomics. 2019 Mar;18(3):576-593. [PubMed:30563849] [doi:10.1074/mcp.TIR118.000943].
[17] Mischnik, M., Sacco, F., Cox, J., Schneider, H.C., Schafer, M., Hendlich, M., Crowther, D., Mann, M. and Klabunde, T. IKAP: A heuristic framework for inference of kinase activities from Phosphoproteomics data. Bioinformatics. 2016 Feb 1;32(3):424-31. [PubMed:26628587] [doi:10.1093/bioinformatics/btv699].
[18] Yang, P., Patrick, E., Humphrey, S.J., Ghazanfar, S., James, D.E., Jothi, R. and Yang, J.Y. KinasePA: Phosphoproteomics data annotation using hypothesis driven kinase perturbation analysis. Proteomics. 2016 Jul;16(13):1868-71. [PubMed:27145998] [PubMed Central:PMC5027648] [doi:10.1002/pmic.201600068].
[19] Wiredja, D.D., Koyuturk, M. and Chance, M.R. The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics. Bioinformatics. 2017 Nov 1;33(21):3489-3491. [PubMed:28655153] [PubMed Central:PMC5860163] [doi:10.1093/bioinformatics/btx415].
[20] Lachmann, A., Xu, H., Krishnan, J., Berger, S.I., Mazloom, A.R. and Ma’ayan, A. ChEA: transcription factor regulation inferred from integrating genome-wide ChIP-X experiments. Bioinformatics. 2010 Oct 1;26(19):2438-44. [PubMed:20709693] [PubMed Central:PMC2944209] [doi:10.1093/bioinformatics/btq466].
[21] Kou, Y., Chen, E.Y., Clark, N.R., Duan, Q., Tan, C.M. and Ma‘ayan, A. (2013). Springer Berlin Heidelberg, Berlin, Heidelberg, pp. 416-430.
[22] Keenan, A.B., Torre, D., Lachmann, A., Leong, A.K., Wojciechowicz, M.L., Utti, V., Jagodnik, K.M., Kropiwnicki, E., Wang, Z. and Ma’ayan, A. ChEA3: transcription factor enrichment analysis by orthogonal omics integration. Nucleic Acids Res. 2019 Jul 2;47(W1):W212-W224. [PubMed:31114921] [PubMed Central:PMC6602523] [doi:10.1093/nar/gkz446].
[23] Berger, S.I., Posner, J.M. and Ma’ayan, A. Genes2Networks: connecting lists of gene symbols using mammalian protein interactions databases. BMC Bioinformatics. 2007 Oct 4;8:372. [PubMed:17916244] [PubMed Central:PMC2082048] [doi:10.1186/1471-2105-8-372].
[24] Lachmann, A. and Ma’ayan, A. KEA: kinase enrichment analysis. Bioinformatics. 2009 Mar 1;25(5):684-6. [PubMed:19176546] [PubMed Central:PMC2647829] [doi:10.1093/bioinformatics/btp026].
[25] Braschi, B., Denny, P., Gray, K., Jones, T., Seal, R., Tweedie, S., Yates, B. and Bruford, E. Genenames.org: the HGNC and VGNC resources in 2019. Nucleic Acids Res. 2019 Jan 8;47(D1):D786-D792. [PubMed:30304474] [PubMed Central:PMC6324057] [doi:10.1093/nar/gky930].
[26] Manning, G., Whyte, D.B., Martinez, R., Hunter, T. and Sudarsanam, S. The protein kinase complement of the human genome. Science. 2002 Dec 6;298(5600):1912-34. [PubMed:12471243] [doi:10.1126/science.1075762].
[27] Miranda‐Saavedra, D. and Barton, G.J. Classification and functional annotation of eukaryotic protein kinases. Proteins. 2007 Sep 1;68(4):893-914. [PubMed:17557329] [doi:10.1002/prot.21444].
[28] Rodgers, G., Austin, C., Anderson, J., Pawlyk, A., Colvis, C., Margolis, R. and Baker, J. Glimmers in illuminating the druggable genome. Nat Rev Drug Discov. 2018 May;17(5):301-302. [PubMed:29348682] [PubMed Central:PMC6309548] [doi:10.1038/nrd.2017.252].
[29] Szklarczyk, D., Gable, A.L., Lyon, D., Junge, A., Wyder, S., Huerta-Cepas, J., Simonovic, M., Doncheva, N.T., Morris, J.H., Bork, P. et al. STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 2019 Jan 8;47(D1):D607-D613. [PubMed:30476243] [doi:10.1093/nar/gky1131].
[30] Lachmann, A., Torre, D., Keenan, A.B., Jagodnik, K.M., Lee, H.J., Wang, L., Silverstein, M.C. and Ma’ayan, A. Massive mining of publicly available RNA-seq data from human and mouse. Nat Commun. 2018 Apr 10;9(1):1366. [PubMed:29636450] [PubMed Central:PMC5893633] [doi:10.1038/s41467-018-03751-6].
[31] Clark, N.R., Hu, K.S., Feldmann, A.S., Kou, Y., Chen, E.Y., Duan, Q. and Ma’ayan, A. The characteristic direction: a geometrical approach to identify differentially expressed genes. BMC Bioinformatics. 2014 Mar 21;15:79. [PubMed:24650281] [PubMed Central:PMC4000056] [doi:10.1186/1471-2105-15-79].
[32] Wang, Z., Monteiro, C.D., Jagodnik, K.M., Fernandez, N.F., Gundersen, G.W., Rouillard, A.D., Jenkins, S.L., Feldmann, A.S., Hu, K.S., McDermott, M.G. et al. Extraction and analysis of signatures from the Gene Expression Omnibus by the crowd. Nat Commun. 2016 Sep 26;7:12846. [PubMed:27667448] [PubMed Central:PMC5052684] [doi:10.1038/ncomms12846].
[33] Wang, Z., Lachmann, A., Keenan, A.B. and Ma’ayan, A. L1000FWD: fireworks visualization of drug-induced transcriptomic signatures. Bioinformatics. 2018 Jun 15;34(12):2150-2152. [PubMed:29420694] [PubMed Central:PMC6454499] [doi:10.1093/bioinformatics/bty060].
[34] Hornbeck, P.V., Zhang, B., Murray, B., Kornhauser, J.M., Latham, V. and Skrzypek, E. PhosphoSitePlus, 2014: mutations, PTMs and recalibrations. Nucleic Acids Res. 2015 Jan;43(Database issue):D512-20. [PubMed:25514926] [PubMed Central:PMC4383998] [doi:10.1093/nar/gku1267].
[35] Garcia-Alonso, L., Ibrahim, M.M., Turei, D. and Saez-Rodriguez, J. Benchmark and integration of resources for the estimation of human transcription factor activities. Genome Res. 2019 Aug;29(8):1363-1375. [PubMed:31340985] [PubMed Central:PMC6673718] [doi:10.1101/gr.240663.118].
[36] Brittain, J. and Darwin, I.F. (2007) Tomcat: the definitive guide.
[37] Mobirise. (2018). 4.8.1 ed, United Kingdom.
[38] Merkel, D. (2014) Docker: lightweight Linux containers for consistent development and deployment. Linux J., 2014, 2.
[39] Langfelder, P. and Horvath, S. (2008) WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics. 2008 Dec 29;9:559. [PubMed:19114008] [PubMed Central:PMC2631488] [doi:10.1186/1471-2105-9-559].
[40] GTEx Consortium. The Genotype-Tissue Expression (GTEx) Project. Biopreserv Biobank. 2015 Oct;13(5):307-8. [PubMed:26484569] [PubMed Central:PMC4692118] [doi:10.1089/bio.2015.29031.hmm].
[41] Shannon, P., Markiel, A., Ozier, O., Baliga, N.S., Wang, J.T., Ramage, D., Amin, N., Schwikowski, B. and Ideker, T. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 2003 Nov;13(11):2498-504. [PubMed:14597658] [PubMed Central:PMC403769] [doi:10.1101/gr.1239303].
[42] Bostock, M., Ogievetsky, V. and Heer, J. D3 Data-Driven Documents. IEEE Trans Vis Comput Graph. 2011 Dec;17(12):2301-9. [PubMed:22034350] [doi:10.1109/TVCG.2011.185]
[43] Miranda-Saavedra, D. and Barton, G.J. Classification and functional annotation of eukaryotic protein kinases. Proteins. 2007 Sep 1;68(4):893-914. [PubMed:17557329] [doi:10.1002/prot.21444].
[44] Oughtred, R., Stark, C., Breitkreutz, B.J., Rust, J., Boucher, L., Chang, C., Kolas, N., O’Donnell, L., Leung, G., McAdam, R. et al. The BioGRID interaction database: 2019 update. Nucleic Acids Res. 2019 Jan 8;47(D1):D529-D541. [PubMed:30476227] [PubMed Central:PMC6324058] [doi:10.1093/nar/gky1079].
[45] Calderone, A., Castagnoli, L. and Cesareni, G. mentha: a resource for browsing integrated protein-interaction networks. Nat Methods. 2013 Aug;10(8):690-1. [PubMed:23900247] [doi:10.1038/nmeth.2561].
[46] Drew, K., Lee, C., Huizar, R.L., Tu, F., Borgeson, B., McWhite, C.D., Ma, Y., Wallingford, J.B. and Marcotte, E.M. Integration of over 9,000 mass spectrometry experiments builds a global map of human protein complexes. Mol Syst Biol. 2017 Jun 8;13(6):932. [PubMed:28596423] [doi:10.15252/msb.20167490].
[47] Zhang, Q.C., Petrey, D., Garzon, J.I., Deng, L. and Honig, B. PrePPI: a structure-informed database of protein-protein interactions. Nucleic Acids Res. 2013 Jan;41(Database issue):D828-33. [PubMed:23193263] [doi:10.1093/nar/gks1231].
[48] Licata, L., Briganti, L., Peluso, D., Perfetto, L., Iannuccelli, M., Galeota, E., Sacco, F., Palma, A., Nardozza, A.P., Santonico, E. et al. MINT, the molecular interaction database: 2012 update. Nucleic Acids Res. 2012 Jan;40(Database issue):D857-61. [PubMed:22096227] [PubMed Central:PMC3244991] [doi:10.1093/nar/gkr930].
[49] Alanis-Lobato, G., Andrade-Navarro, M.A. and Schaefer, M.H. HIPPIE v2.0: enhancing meaningfulness and reliability of protein-protein interaction networks. Nucleic Acids Res. 2017 Jan 4;45(D1):D408-D414. [PubMed:27794551] [doi:10.1093/nar/gkw985].
[50] McDowall, M.D., Scott, M.S. and Barton, G.J. PIPs: human protein-protein interaction prediction database. Nucleic Acids Res. 2009 Jan;37(Database issue):D651-6. [PubMed:18988626] [doi:10.1093/nar/gkn870].
[51] Murakami, Y. and Mizuguchi, K. (2017), 2017 International Conference on Intelligent Informatics and Biomedical Sciences. IEEE, Okinawa, Japan.
[52] Cheng, F., Jia, P., Wang, Q. and Zhao, Z. Quantitative network mapping of the human kinome interactome reveals new clues for rational kinase inhibitor discovery and individualized cancer therapy. Oncotarget. 2014 Jun 15;5(11):3697-710. [PubMed:25003367] [PubMed Central:PMC4116514] [doi:10.18632/oncotarget.1984].
[53] Licata, L. and Orchard, S. The MIntAct Project and Molecular Interaction Databases. Methods Mol Biol. 2016;1415:55-69. [PubMed:27115627] [doi:10.1007/978-1-4939-3572-7_3].
[54] Orchard, S., Ammari, M., Aranda, B., Breuza, L., Briganti, L., Broackes-Carter, F., Campbell, N.H., Chavali, G., Chen, C., del-Toro, N. et al. The MIntAct project–IntAct as a common curation platform for 11 molecular interaction databases. Nucleic Acids Res. 2014 Jan;42(Database issue):D358-63. [PubMed:24234451] [doi:10.1093/nar/gkt1115].
[55] Clerc, O., Deniaud, M., Vallet, S.D., Naba, A., Rivet, A., Perez, S., Thierry-Mieg, N. and Ricard-Blum, S. MatrixDB: integration of new data with a focus on glycosaminoglycan interactions. Nucleic Acids Res. 2019 Jan 8;47(D1):D376-D381. [PubMed:30371822] [PubMed Central:PMC6324007] [doi:10.1093/nar/gky1035].
[56] Salwinski, L., Miller, C.S., Smith, A.J., Pettit, F.K., Bowie, J.U. and Eisenberg, D. The Database of Interacting Proteins: 2004 update. Nucleic Acids Res. 2004 Jan 1;32(Database issue):D449-51. [PubMed:14681454] [PubMed Central:PMC308820] [doi:10.1093/nar/gkh086].
[57] Keshava Prasad, T.S., Goel, R., Kandasamy, K., Keerthikumar, S., Kumar, S., Mathivanan, S., Telikicherla, D., Raju, R., Shafreen, B., Venugopal, A. et al. Human Protein Reference Database–2009 update. Nucleic Acids Res. 2009 Jan;37(Database issue):D767-72. [PubMed:18988627] [doi:10.1093/nar/gkn892].
[58] Bader, G.D., Betel, D. and Hogue, C.W. BIND: the Biomolecular Interaction Network Database. Nucleic Acids Res. 2003 Jan 1;31(1):248-50. [PubMed:12519993] [PubMed Central:PMC165503] [doi:10.1093/nar/gkg056].
[59] Pagel, P., Kovac, S., Oesterheld, M., Brauner, B., Dunger-Kaltenbach, I., Frishman, G., Montrone, C., Mark, P., Stumpflen, V., Mewes, H.W. et al. The MIPS mammalian protein-protein interaction database. Bioinformatics. 2005 Mar;21(6):832-4. [PubMed:15531608] [doi:10.1093/bioinformatics/bti115].
[60] Guldener, U., Munsterkotter, M., Oesterheld, M., Pagel, P., Ruepp, A., Mewes, H.W. and Stumpflen, V. MPact: the MIPS protein interaction resource on yeast. Nucleic Acids Res. 2006 Jan 1;34(Database issue):D436-41. [PubMed:16381906] [PubMed Central:PMC1347366] [doi:10.1093/nar/gkj003].
[61] Fabregat, A., Jupe, S., Matthews, L., Sidiropoulos, K., Gillespie, M., Garapati, P., Haw, R., Jassal, B., Korninger, F., May, B. et al. The reactome pathway knowledgebase. Nucleic Acids Res. 2020 Jan 8;48(D1):D498-D503. [PubMed:31691815] [PubMed Central:PMC5753187] [doi:10.1093/nar/gkz1031].
[62] Zhang, Q.C., Petrey, D., Deng, L., Qiang, L., Shi, Y., Thu, C.A., Bisikirska, B., Lefebvre, C., Accili, D., Hunter, T. et al. Structure-based prediction of protein-protein interactions on a genome-wide scale. Nature. 2012 Oct 25;490(7421):556-60. [PubMed:23023127] [PubMed Central:PMC3482288] [doi:10.1038/nature11503].
[63] Hu, J., Rho, H.S., Newman, R.H., Zhang, J., Zhu, H. and Qian, J. PhosphoNetworks: a database for human phosphorylation networks. Bioinformatics. 2014 Jan 1;30(1):141-2. [PubMed:24227675] [PubMed Central:PMC3866559] [doi:10.1093/bioinformatics/btt627].
[64] Dinkel, H., Chica, C., Via, A., Gould, C.M., Jensen, L.J., Gibson, T.J. and Diella, F. Phospho.ELM: a database of phosphorylation sites–update 2011. Nucleic Acids Res. 2011 Jan;39(Database issue):D261-7. [PubMed:21062810] [doi:10.1093/nar/gkq1104].
[65] Qin, G.M., Li, R.Y. and Zhao, X.M. PhosD: inferring kinase-substrate interactions based on protein domains. Bioinformatics. 2017 Apr 15;33(8):1197-1204. [PubMed:28031187] [doi:10.1093/bioinformatics/btw792].
[66] Hu, J., Rho, H.S., Newman, R.H., Hwang, W., Neiswinger, J., Zhu, H., Zhang, J. and Qian, J. Global analysis of phosphorylation networks in humans. Biochim Biophys Acta. 2014 Jan;1844(1 Pt B):224-31. [PubMed:23524292] [PubMed Central:PMC3815481] [doi:10.1016/j.bbapap.2013.03.009].
[67] Newman, R.H., Hu, J., Rho, H.S., Xie, Z., Woodard, C., Neiswinger, J., Cooper, C., Shirley, M., Clark, H.M., Hu, S. et al. Construction of human activity-based phosphorylation networks. Mol Syst Biol. 2013;9:655. [PubMed:23549483] [PubMed Central:PMC3658267] [doi:10.1038/msb.2013.12].
[68] Carithers, L.J. and Moore, H.M. The Genotype-Tissue Expression (GTEx) Project. Biopreserv Biobank. 2015 Oct;13(5):307-8. [PubMed:26484569] [PubMed Central:PMC4692118] [doi:10.1089/bio.2015.29031.hmm].
[69] Pita-Juarez, Y., Altschuler, G., Kariotis, S., Wei, W., Koler, K., Green, C., Tanzi, R.E. and Hide, W. The Pathway Coexpression Network: Revealing pathway relationships. PLoS Comput Biol. 2018 Mar 19;14(3):e1006042. [PubMed:29554099] [PubMed Central:PMC5875878] [doi:10.1371/journal.pcbi.1006042].
[70] Wang, Z., Clark, N.R. and Ma’ayan, A. Dynamics of the discovery process of protein-protein interactions from low content studies. BMC Syst Biol. 2015 Jun 6;9:26. [PubMed:26048415] [PubMed Central:PMC4456804] [doi:10.1186/s12918-015-0173-z].
[71] Scott, M.S. and Barton, G.J. Probabilistic prediction and ranking of human protein-protein interactions. BMC Bioinformatics. 2007 Jul 5;8:239. [PubMed:17615067] [PubMed Central:PMC1939716] [doi:10.1186/1471-2105-8-239].
